src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,20 @@
+
+.dg_summary-list__item {
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  background-color: $gray-lightest;
+  color: $gray;
+  padding: $margin-large;
+  h3 {
+    margin-top: 0;
+    color: $gray;
```

Comment:
Based on the [xd](https://xd.adobe.com/spec/ad47a25b-c580-4389-756b-04b95b9d8a00-db76/screen/517b87a9-0607-484f-a6a4-fff046698f4f/18-Example-Adoptable-Pets-Detail) document it looks like this should be black.

![image](https://user-images.githubusercontent.com/1933400/65876418-2ee61000-e357-11e9-86d0-7117ee94f49c.png)

Reply:
done